.github/workflows/ros2foxy.yml

```diff
@@ -36,10 +36,10 @@ jobs:
         # TODO(ros-tooling/setup-ros-docker#7): calling chown is necessary for now
         sudo chown -R rosbuild:rosbuild "$HOME" .
     - name: Scan using git-secrets
-      uses: aws-robotics/aws-robomaker-github-actions/git-secrets-scan-action@2.5.0
+      uses: aws-robotics/aws-robomaker-github-actions/git-secrets-scan-action@2.5.1
```

Comment:
I don't think git-secrets-scan-action needs to get bumped

Reply:
we don't really - this action should be unchanged.

Reply:
> Should we also bump `robomaker-sample-app-ci` for `ros1` and `ros2`?

I don't think so, since those are building fine and don't required the updated action.